pkgs/development/lua-modules/overrides.nix

```
@@ -671,17 +677,19 @@ with prev;
     propagatedBuildInputs = oa.propagatedBuildInputs ++ [ magic-enum sol2 ];
```

suggestion: magic-enum is no longer needed as a propagated build input.
To make sure the linking of tomlplusplus succeeds, we need to add it to the `nativeBuildInputs`.

question: Do you know why it fails with `tomlplusplus::tomlplusplus`?

```suggestion
    nativeBuildInputs = oa.nativeBuildInputs ++ [ tomlplusplus ];
    propagatedBuildInputs = oa.propagatedBuildInputs ++ [ sol2 ];
```